src/libs/ReportUtils.js

```diff
@@ -1257,15 +1257,14 @@ function getIOUTotal(report, iouReports = {}) {
 /**
  * @param {Object} report
  * @param {String} report.iouReportID
- * @param {String} currentUserLogin
  * @param {Object} iouReports
  * @returns {Boolean}
  */
-function isIOUOwnedByCurrentUser(report, currentUserLogin, iouReports = {}) {
+function isIOUOwnedByCurrentUser(report, iouReports = {}) {
```

Comment:
`currentUserLogin` is already declared in the upper scope - https://github.com/Expensify/App/blob/70f5212c92185118e5ff9977e1c4dc856f5b4e06/src/libs/ReportUtils.js#L42